api-docs/rst/dev-guide/release-notes/as_20151204.rst

```diff
@@ -0,0 +1,34 @@
+API v1.0 release, December 08, 2015
+-----------------------------------
+
+What's new
+~~~~~~~~~~
+
+*  Added a new :ref:`converge endpoint <trigger-convergence>`. You can submit a
+   **POST** request to this endpoint to trigger convergence. Kindly check
+   :ref:`convergence <convergence>`.
+
```

Comment:
Added a new :ref:`Trigger convergence <post-converge-v1.0-tenantid-groups-groupid>` API operation that you can use to adjust scaling group configuration when a group diverges from the launch configuration specification. For more information, see :ref:`convergence <convergence-concept> in the *Concepts* section.